wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
-<img src="Options Base.jpg" title="Options menu (yes, that sidebar at left)" alt="Options menu (yes, that sidebar at left)" width="300" />
-
-Options
-=======
-
-`Restart osu! if anything isn't working as intended (Background Dim is a prime offender). `
+# Options
 
 Click **Options** or press **O** in the Intro Screen to access the Options sidebar and change the way osu! behaves.
```

Comment:
replace `**O**` with `` `O` `` (this is a key on the keyboard, which are to be using code)